packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1129,6 +1130,10 @@ oie.yubikey.challenge.title = Verify with YubiKey
 oie.yubikey.description = Insert the YubiKey into a USB port and tap it to generate a verification code.
 oie.yubikey.passcode.label = Insert then tap your YubiKey
 
+## Custom App Authenticator
+# {0} refers to the display name of the authenticator "My Custom Auth"
```

Comment:
nit: I understand it as "My Custom Auth" is the authenticator name but I believe it can be anything. Can it be changed to `# {0} refers to the display name of the authenticator like "My Custom Auth"`

Reply:
good point, I think we can even be just generic about this, without a name. It still gives the context to the translators
`# {0} is the display name of the authenticator`
We already use this around [line 490](https://github.com/okta/okta-signin-widget/blob/33291ef94d29bbc9543a0b3fe8b8aa36f1549b54/packages/%40okta/i18n/src/properties/login.properties#L494)